framework/build.sh

```diff
@@ -7,25 +7,27 @@ cd build
 
 cmake \
   -G Ninja \
-  -DUSE_SYSTEM_EIGEN=ON \
-  -DUSE_JEMALLOC=OFF \
-  -DENABLE_OPENGL=OFF \
-  -DENABLE_DOCS=OFF \
-  -DENABLE_MANTIDPLOT=FALSE \
-  -DENABLE_WORKBENCH=FALSE \
-  -DENABLE_OPENCASCADE=FALSE \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DUSE_JEMALLOC=ON \
+  -DUSE_CCACHE=OFF \
   -DPython_EXECUTABLE="$CONDA_PREFIX/bin/python" \
   -DCMAKE_PREFIX_PATH=$CONDA_PREFIX \
   -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX \
-  -DCMAKE_SKIP_INSTALL_RPATH=ON \
+  -DCMAKE_OSX_DEPLOYMENT_TARGET=$OSX_VERSION \
+  -DHDF5_ROOT=$CONDA_PREFIX \
+  -DCMAKE_OSX_SYSROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX$OSX_VERSION.sdk" \
+  -DUSE_SYSTEM_EIGEN=OFF\
+  -DENABLE_MANTIDPLOT=OFF\
+  -DENABLE_WORKBENCH=OFF\
+  -DENABLE_OPENGL=OFF\
+  -DENABLE_DOCS=OFF\
+  -DENABLE_OPENCASCADE=OFF\
+  -DUSE_PYTHON_DYNAMIC_LIB=OFF\
   ..
 
+# cmake -LA
 cmake --build .
 cmake --build . --target install
 
-# move mantid
-python_site_pkg_path=`python -c "from __future__ import print_function; import h5py, os; opd=os.path.dirname; print(opd(opd(h5py.__file__)))"`
-echo $python_site_pkg_path
-
-mv ${CONDA_PREFIX}/lib/mantid $python_site_pkg_path/
-mv ${CONDA_PREFIX}/lib/mantid-*-py*.egg-info $python_site_pkg_path/
+mv ${CONDA_PREFIX}/lib/mantid ${SP_DIR}
```

Comment:
Is `SP_DIR` defined by conda build?
Do we not need to move the egg files too?

Reply:
Yes, that's listed here https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html

I'm not sure what the purpose of doing that is, but we should be able to enable if necessary - maybe @peterfpeterson  can comment.

Reply:
It looks like the new version moves everything (`.egg` included). The old appears redundant, but maybe I'm misunderstanding.